sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -412,6 +419,63 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
+   * Register a Java UDF class
+   * @param name
+   * @param className
+   * @param returnType
+   */
+  def registerJava(name: String, className: String, returnType: DataType): Unit = {
```

Comment:
+ 1 for update the scaladoc. `Register a Java UDF class` doesn't exactly convey the meaning of this function.